app/server/api/vendor_payout_api.py

```diff
@@ -26,8 +26,10 @@ def post(self):
         # Process post data
         post_data = request.get_json()
         account_ids = []
+        relist_existing = True
         if post_data:
             account_ids = post_data.get('accounts', [])
+            relist_existing = post_data.get('relist_existing', True)
 
         if not isinstance(account_ids, list):
 
```

Comment:
IF a CONTRACT or ORGANISATION transfer account ID is passed back, then the `vendor.primary_user.has_vendor_role` on Line 44 throws a 500 error, as it is not caught by the SQL query on Line 42
```
File "/Users/tristancole/PycharmProjects/SempoBlockchain/app/server/api/vendor_payout_api.py", line 44, in post
    if not vendor.primary_user.has_vendor_role:
AttributeError: 'NoneType' object has no attribute 'has_vendor_role'
```